index.html

```diff
@@ -1102,6 +1130,17 @@ <h2 id="about">About</h2>
       <p>Made with 💛 by <a href="https://vlaservi.ch" target="_blank">Rhyne</a> and some <a href="https://github.com/rhyneav/papercss/graphs/contributors">fantastic contributors</a>!</p>
     </div>
   </div>
+  <script>
+    /* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
+    function myFunction() {
```

Comment:
can this function have a better name, "ToggleNavBarVisibility"

Reply:
Thank you for feedback, I will rewrite it